services/summary.go

```diff
@@ -536,3 +537,40 @@ func (srv *SummaryService) getProjectLabelsReverseResolver(user *models.User) mo
 		return projectStrings
 	}
 }
+
+func (srv *SummaryService) GetDailyProjectStats(from, to time.Time, user *models.User) ([]models.DailyProjectStat, error) {
```

Comment:
Minor one: we usually return lists of pointers (e.g. `[]*models.DailyProjectStat`). While it doesn't really make any difference, I'd still prefer to stick to that here as well, just for consistency.